calc/src/move.ts

```diff
@@ -89,14 +89,18 @@ export class Move implements State.Move {
       });
     } else {
       if (data.multihit) {
-        if (typeof data.multihit === 'number') {
-          this.hits = data.multihit;
-        } else if (options.hits) {
-          this.hits = options.hits;
+        if (data.multiaccuracy && typeof data.multihit === 'number') {
+          this.hits = options.hits || data.multihit;
         } else {
-          this.hits = (options.ability === 'Skill Link' || options.item === 'Grip Claw')
-            ? data.multihit[1]
-            : data.multihit[0] + 1;
+          if (typeof data.multihit === 'number') {
+            this.hits = data.multihit;
+          } else if (options.hits) {
+            this.hits = options.hits;
+          } else {
+            this.hits = (options.ability === 'Skill Link' || options.item === 'Grip Claw')
```

Comment:
Unrelated to the PR, but I have no idea what Grip Claw is doing here; it doesn't have any effect on multihit moves. Looks like a very old mechanics inaccuracy introduced a [few years ago](https://github.com/smogon/damage-calc/commit/1f95cd0bc456ed2903b9d578a391fbcd3feeb52d). I'll correct it.

Reply:
resolved in #510